test/basic.js

```diff
@@ -3,49 +3,75 @@ var create = require('./helpers/create')
 
 tape('basic put/get', function (t) {
   var db = create.one()
-  db.put('/hello', 'world', function (err, _node) {
+  db.put('hello', 'world', function (err) {
```

Comment:
I'm feeling concerned about all of the simultaneous test changes + implementation changes. For API compatibility I like the idea of keeping the original (even if breaking) tests around so we can iterate on the codebase until 100% of the pre-`next` tests are passing again.